src/mlpack/bindings/python/print_output_processing.hpp

```diff
@@ -167,9 +170,10 @@ void PrintOutputProcessing(
  */
 template<typename T>
 void PrintOutputProcessing(
+    util::Params& params,
     util::ParamData& d,
     const size_t indent,
-    const bool onlyOutput,
+    const bool onlyOutput, 
```

Comment:
```suggestion
    const bool onlyOutput,
```
No need for an extra space. :smiel: